alvr/xtask/flatpak/com.valvesoftware.Steam.Utility.alvr.desktop

```diff
@@ -1,5 +1,5 @@
 [Desktop Entry]
-Version=21.0
+Version=1.0
```

Comment:
hmmm. I get the intentions, but I still don't like it

Reply:
I told @failboat78 to do this. We currently don't have a version for the launcher, which should be separate than the server version. We will make this overridable at a later point